pkgs/applications/system/linux-enable-ir-emitter/default.nix

```
@@ -0,0 +1,60 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchurl
+, opencv
+, pkg-config
+, python3
+}:
+let
+  pythonInUse =
+    python3.withPackages
+      (p: [ p.face_recognition (p.opencv4.override { enableGtk3 = true; }) ]);
```

```suggestion
  python = python3.withPackages (p: [
    p.face_recognition
    (p.opencv4.override { enableGtk3 = true; })
  ]);
```
some nit formatting and we would call this python just python or py

---

Sounds good.